mscore/musescore.cpp

```diff
@@ -541,8 +541,8 @@ MuseScore::MuseScore()
 #endif
       viewModeCombo->setAccessibleName(tr("View Mode"));
       viewModeCombo->setFixedHeight(preferences.iconHeight + 8);  // hack
-      viewModeCombo->addItem(tr("Page View"));
-      viewModeCombo->addItem(tr("Continuous View"));
+      viewModeCombo->addItem(tr("Page Mode"));
```

Comment:
It was discussed on forum - see http://musescore.org/en/node/42616.  I'm ambivalent.  To me, "mode" implies something about what you can or cannot *do*, not just whjat it looks like.  I think "view" makes more sense.